src/app/lightning/get-balances.ts

```diff
@@ -41,6 +41,17 @@ export const getOffChainBalance = async (): Promise<Satoshis | ApplicationError>
     },
   })
 
+export const getOffChainChannelBalances = async () =>
+  cache.getOrSet({
```

Comment:
I don't think this make sense to cache this value

Reply:
if lnd is offline, the swap will fail anyway.

Reply:
cache removed